lib/mayaUsd/fileio/writeJobContext.cpp

```diff
@@ -176,14 +180,14 @@ SdfPath UsdMayaWriteJobContext::ConvertDagToUsdPath(const MDagPath& dagPath) con
         path = path.GetParentPath();
     }
 
-    path = _GetRootOverridePath(mArgs, path);
+    path = _GetRootOverridePath(mArgs, path, false, true);
```

Comment:
Can you comment each argument so we can figure out which is which at the call site?  E.g.
```suggestion
    path = _GetRootOverridePath(mArgs, path, /* modelRootOverride = */ false, /* rootMap = */ true);
```
Same at line 190.